include/gatekeeper_fib.h

```diff
@@ -197,6 +197,8 @@ struct ip_prefix {
 
 struct gk_config;
 
+typedef int (*gk_fib_dump_entry_handler)(void *fib_dump_entry, void *arg);
```

Comment:
1. Move the declaration of type `gk_fib_dump_entry_handler` just before the declaration of `list_fib_entries()` in this file.

2. The first parameter of `gk_fib_dump_entry_handler` is always of the type `struct gk_fib_dump_entry *`, so why is `void *` being used here?